pkg/loki/config_wrapper_test.go

```diff
@@ -1331,3 +1331,17 @@ storage_config:
 	})
 
 }
+
+func Test_repcliationFactor(t *testing.T) {
+	t.Run("replication factor is applied when using memberlist", func(t *testing.T) {
+		yamlContent := `---
+memberlist:
```

Comment:
nit (first line not necessary)
```suggestion
		yamlContent := `memberlist:
```